pkg/instrumentation/categories.go

```diff
@@ -66,6 +67,11 @@ func determineCategoryFromArgs(args []string, knownCommands []string, flagsAllow
 			if slices.Contains(flagsAllowList, flagName) {
 				flags = append(flags, strings.ToLower(flagName))
 			}
+
+			// a shorthand variant for debug
```

Comment:
nitpick: do we really need this comment?

Reply:
Removed